src/export_figure.py

```diff
@@ -61,6 +61,7 @@ def on_response(dialog, response):
         dialog = Gtk.FileDialog()
         dialog.set_initial_name(f"{filename}.{file_suffix}")
         dialog.set_accept_label(_("Export"))
-        dialog.set_filters(utilities.create_file_filters([(fmt, file_suffix)]))
+        dialog.set_filters(utilities.create_file_filters(
+            [(fmt, [file_suffix])]))
```

Comment:
With the ability to add multiple formats I'd use all available ones in this case.
On line 45 we use just the first, but we can use just the first for the initial filename.